aspnetcore/migration/inc/start.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Get started with incremental ASP.NET to ASP.NET Core migration
+description: Get started with incremental ASP.NET to ASP.NET Core migration
+author: rick-anderson
+ms.author: riande
+monikerRange: '>= aspnetcore-6.0'
+ms.date: 11/9/2022
+ms.topic: article
+ms.prod: aspnet-core
+uid: migration/inc/start
+---
+
+# Get started with incremental ASP.NET to ASP.NET Core migration
+
+In order to start a large migration, we recommend setting up a ASP.NET Core app that will proxy to the original .NET Framework app. This set up will look like this:
+
+![final pic](~/migration/inc/overview/static/final.png)
+
+To understand how this is helpful in the migration process, see [Incremental ASP.NET to ASP.NET Core migration](xref:migration/inc/overview). The rest of this article provides the steps to set this up and how to proceed with an incremental migration.
+
+## Set up ASP.NET Core Project
+
+1. Install the [experimental Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=WebToolsTeam.aspnetprojectmigrations) that helps configure the solution.
+2. Right click the ASP.NET Framework app and select **Migrate Project**:
+   ![Migrate Menu](~/migration/inc/start/static/migrate_menu.png)
+1. This will open a menu that will offer to start a migration. Click the link to begin:
+   ![Migrate Options](~/migration/inc/start/static/migrate_options.png)
+1. A wizard will now appear that allows you to create a new project or select an existing project.
+   ![Migrate Wizard](~/migration/inc/start/static/migrate_wizard.png)
+1. After completing the wizard, you have an ASP.NET Core project that proxies requests that do not exist there onto the ASP.NET Framework app.
+
+## Upgrade supporting libraries
+
+If you have supporting libraries in your solution that you will need to use, they should be upgraded to support .NET 6 or later<!--Review -->. [Upgrade Assistant](https://github.com/dotnet/upgrade-assistant) is a great tool for this.
```

Comment:
Review

Reply:
The adapters support .NET 6+ at runtime, so this makes sense.